include/fmt/core.h

```diff
@@ -189,11 +189,21 @@
 #    define FMT_API __declspec(dllexport)
```

Comment:
It should be like this:
```
#if !defined(FMT_HEADER_ONLY) && defined(_WIN32)
#  ifdef FMT_EXPORT
#    define FMT_API __declspec(dllexport)
#    define FMT_EXTERN_TEMPLATE_API FMT_API
#  elif defined(FMT_SHARED)
#    define FMT_API __declspec(dllimport)
#    define FMT_EXTERN_TEMPLATE_API FMT_API
#  endif
#endif
```

With the addition
```
#    define FMT_EXTERN_TEMPLATE_API FMT_API
```
the MinGW shared build is fixed.

Reply:
@NikitaFeodonit, if you get an error in MinGW please open a new issue (or a pull request) with more details.